docs/content/stable/yugabyte-platform/back-up-restore-universes/back-up-universe-data.md

```diff
@@ -136,8 +138,50 @@ s3://user_bucket/some/sub/folders
 
 A backup set consists of a successful full backup, and (if incremental backups were taken) one or more consecutive successful incremental backups. The backup set can be used to restore a database at the point in time of the full and/or incremental backup, as long as the chain of good incremental backups is unbroken. Use the creation time to identify increments that occurred after a full backup.
 
-Although you can move a backup from its location, for a successful restore, none of the sub-components and folder names can be modified (from the universe UUID folder on down).
+Although you can move a backup from its location, for a successful restore, only the bucket can be changed, and none of the sub-components and folder names can be modified (from the sub-directories on down). For examples, see [Moving backups between buckets](#moving-backups-between-buckets).
```

Comment:
would move this par to new moving backups section as intro

Reply:
Had it this way originally. will revert the change.